### ISSUE TITLE
Change link in the running ingestions logs

### ISSUE THREAD

**Issue by marcos-lg** [gbif/registry-console#491]
In the `log` link of the running ingestions here https://registry.gbif.org/monitoring/running-ingestions it has this query to search for the logs in kibana:

```
datasetKey.keyword:"{datasetKey}" AND attempt:"{attempt}"
```

It seems that it's filtering by `datasetKey` but it doesn't highlight it in yellow like it does with the `attempt`. But if we do the search without the `.keyword` in the `datasetKey` parameter it highlights it:

```
datasetKey:"{datasetKey}" AND attempt:"{attempt}"
```

Can we change it?

**Comment by MortenHofft** [comment]
Yes we can, but I'm not entirely sure it is a good idea. Dataset is indexed as text perhaps? So it wont be a precise match then,
